.env

```diff
@@ -0,0 +1,6 @@
+MIKRO_ORM_DB_NAME=node_gmp
```

Comment:
Usually we don't commit `.env` file on a real project, instead we commit into git `.env.example` file with fake values to make our security strengthener